modules.d/95udev-rules/module-setup.sh

```diff
@@ -68,10 +67,7 @@ install() {
         "${udevdir}"/ata_id \
         "${udevdir}"/cdrom_id \
         "${udevdir}"/create_floppy_devices \
-        "${udevdir}"/firmware.sh \
         "${udevdir}"/firmware \
```

Comment:
Isn't the `/lib/udev/firmware` binary also deprecated since https://github.com/systemd/systemd/commit/be2ea723b1d0 ?